src/hooks/proxy-state/index.ts

```diff
@@ -7,10 +7,10 @@ import {
  * @name useProxyValue
  * @description In case of array or object, set() does not work even if the original value is modified unless reallocated.
  */
-export function useProxyValue<T = any, Props = any, Emits = SetupContext['emit']>(
+export function useProxyValue<T = any>(
     name: string,
-    props: Props,
-    emit: Emits,
```

Comment:
😮

Reply:
😮

Reply:
This is to move the responsibility of strict select mode to the filterable dropdown rather than the context menu component.